development/text-editor-app/components/text-editor/Example/Example.tsx

```diff
@@ -17,16 +21,37 @@ const Example = () => {
 		}
 		return null;
 	}
-	function OnChange(editorRef: any, payload: any) {
-		console.log(payload, "payload", editorRef, "editorRef");
+
+	function OnChange(_editorRef: LexicalEditor | null, payload: any) {
```

Comment:
Convention for functions is that they should always start with small letter. Please follow convention for functions.